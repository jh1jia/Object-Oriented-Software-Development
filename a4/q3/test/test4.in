stdin dropfirst 3 count 1 doublewords doublewords count 1 count 2 allcaps
Hello Worlds
