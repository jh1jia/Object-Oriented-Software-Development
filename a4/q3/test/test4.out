LO
LO
LO
LO
LDS
LDS
LDS
LDS
